README.md

```diff
@@ -16,6 +17,7 @@ Documentation can be found on the [Elastic website][elastic]:
 [auditbeat]: https://www.elastic.co/guide/en/beats/auditbeat/current/running-on-docker.html
 [filebeat]: https://www.elastic.co/guide/en/beats/filebeat/current/running-on-docker.html
 [heartbeat]: https://www.elastic.co/guide/en/beats/heartbeat/current/running-on-docker.html
+[journalbeat]: https://www.elastic.co/guide/en/beats/journalbeat/current/running-on-docker.html
```

Comment:
404 :)

Reply:
I've created a PR here that adds the docker content: https://github.com/elastic/beats/pull/9386

Reply:
Are there any news on that?
Is there already a docker image available for journalbeat?
Tried to pull some versions, but no success and unfortunately there is no latest at any beat..

Reply:
@dedemorton It is correct. Its release is scheduled for 6.6.

Reply:
6.6 is release, but still no doc and images disapeared.

Reply:
Hi @tardyp,

You can pull this Docker image from our registry:
```
docker pull docker.elastic.co/beats/journalbeat:6.6.0
6.6.0: Pulling from beats/journalbeat
a02a4930cb5d: Already exists
3e1d11318120: Pull complete
6a2daf26539e: Pull complete
3c8eb3bcd46e: Pull complete
3101fedd1308: Pull complete
18021bedd90a: Pull complete
941f3bf14a14: Pull complete
Digest: sha256:6288b53f985bf9e705ec87668629976596abf0d7714f6960f207fa6fa9ad068f
Status: Downloaded newer image for docker.elastic.co/beats/journalbeat:6.6.0
```
I'm doing the work to have this docker image available in Dockerhub and displayed on https://www.docker.elastic.co/
Sorry for the trouble.

Reply:
https://www.docker.elastic.co/ has been updated to include `journalbeat` and the image is available on docker hub now https://cloud.docker.com/u/elastic/repository/docker/elastic/journalbeat.

Reply:
super cool thanks! I have been able to deploy the 6.6.0 image and replace mheese/journalbeat with elastic/journatbeat !

Reply:
The doc additions have already been merged into master. I will cherry-pick them into the 6.6 branch today.

Reply:
Actually I did backport it. It just needs to be reviewed so I can merge it: https://github.com/elastic/beats/pull/10386